message-markdown.md

```diff
@@ -0,0 +1,44 @@
+# Markdown in messages
+
+Deltachat is going to support the following subset of markdown:
+
+### `**bold**`
+
+### `__underline__`
+
+### `~~strikethrough~~`
+
+### `https://delta.chat` - Urls
```

Comment:
to be honest, i have not seen urls being formatted this way. in fact, eg. github _prevents_ urls being clickable when surrounded by backticks.
i suggest to use `<https://example.org>` instead

Reply:
The backticks here are meant as a document style thing - so that the thing that is written in there is shown as plaintext.